### PR TITLE
Create keybinding handler without registering a listener

### LINKED ISSUE #132: ability to create handler without adding event listener
In some cases, it is useful to just have access to the handler _without_ it being registered as a listener. One use case is where you do not yet have access to the element instance, but wish to eagerly create a handler. 

Something like:

```ts
import { createListener } from "tinykeys";

const handler = createListener(shortcuts)

// later
someElement.addEventListener("keydown", handler)
```

Is this something you would support being added? Happy to make a PR if so

### PR DESCRIPTION
**Pull request by WickyNilliams**
fixes #132

### CODE REVIEW

**Comment by WickyNilliams** [comment]
BTW prettier suggested some formatting when i run `npm run check:format`, but i did not include it in this PR, as i thought i'd leave that to you

File: src/tinykeys.ts [hunk 1]
```diff
@@ -132,24 +130,19 @@ function match(event: KeyboardEvent, press: KeyBindingPress): boolean {
...
  */
-export default function keybindings(
-	target: Window | HTMLElement,
-	keyBindingMap: KeyBindingMap,
-	options: KeyBindingOptions = {},
-): () => void {
-	let timeout = options.timeout ?? DEFAULT_TIMEOUT
-	let event = options.event ?? DEFAULT_EVENT
-
+export function createKeybindingsHandler(keyBindingMap: KeyBindingMap, timeout: number = DEFAULT_TIMEOUT): EventListener {
```
Comment (jamiebuilds) [inline src/tinykeys.ts:137]: Could you make this second parameter a `options` object?

```ts
export interface KeyBindingHandlerOptions {
  // ...
  timeout?: number
}
export interface KeyBindingOptions extends KeyBindingsHandlerOptions {
  // ...
  event?: "keydown" | "keyup"
}
```

Comment (WickyNilliams) [reply src/tinykeys.ts]: Sure. Good idea. Made the change and squashed into one commit

Comment (WickyNilliams) [reply src/tinykeys.ts]: And this time i let prettier do its thing on the ts file, since there was inconsistent use of spaces for indentation

**Pull request merged**